roles/deploy/tasks/nfs.yml

```
@@ -0,0 +1,24 @@
+---
+
+- name: install nfs-utils
+  yum: name=nfs-utils state=latest
```

Use native YAML syntax here rather than the shortened form. It's generally easier to read, plays nicer with syntax highlighting, and is the convention we follow in most projects:

```suggestion
- name: Install nfs-utils
  yum:
    name: nfs-utils
    state: latest
```